cmd/vicadmin/logs.go

```diff
@@ -86,12 +90,33 @@ func (r dlogReader) open() (entry, error) {
 	return newBytesEntry(name+".log", buf.Bytes()), nil
 }
 
+// logFiles has a potential race condition since logrotation can rotate files this moment of time.
+// however, the likely hood of this race is so low that it doesn't worth investing the time to do
```

Comment:
haha, in this case it is just a collection of log data, the most recent logs will be collected in any case.